Let automake know it must clean stamp-vti version.texi files. 
It makes it possible to build grep twice in a row.

Doc about automake clean variables:
http://www.gnu.org/software/automake/manual/html_node/Clean.html

MOSTLYCLEANFILES = stamp-vti version.texi

--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -23,6 +23,6 @@
 
 EXTRA_DIST = grep.in.1
-CLEANFILES = grep.1 egrep.1 fgrep.1
+CLEANFILES = grep.1 egrep.1 fgrep.1 stamp-vti version.texi
 
 grep.1: grep.in.1
 	$(AM_V_GEN)rm -f $@-t $@
